app/routes/app/$type/s/$id.tsx

```diff
@@ -72,35 +72,33 @@ export default function ChallengeSubmission() {
           <ReviewQuestionDrawerButton />
         </section>
         <section className="flex flex-col space-y-7 pb-24">
-          <div className="flex flex-wrap gap-x-8 gap-y-4">
-            <Detail>
-              <DetailItem title="Author">
-                <UserBadge url="u/id" address="0x983110309620D911731Ac0932219af06091b6744" balance={200} />
-              </DetailItem>
-              <DetailItem title="Created">
-                <Badge>{fromNow(submission.createdAt.toString())}</Badge>
-              </DetailItem>
-              <DetailItem title="Overall Score">
-                <Badge className={clsx(SCORE_COLOR_SECONDARY[submission.scoreStatus as Grade], "pl-0")}>
-                  <Badge className={clsx(SCORE_COLOR[submission.scoreStatus as Grade], "mr-2")}>
-                    {submission.scoreStatus}
-                  </Badge>
-                  <span>80</span>
+          <Detail className="flex flex-wrap gap-x-8 gap-y-4">
+            <DetailItem title="Author">
+              <UserBadge url="u/id" address="0x983110309620D911731Ac0932219af06091b6744" balance={200} />
+            </DetailItem>
+            <DetailItem title="Created">
+              <Badge>{fromNow(submission.createdAt.toString())}</Badge>
+            </DetailItem>
+            <DetailItem title="Overall Score">
+              <Badge className={clsx(SCORE_COLOR_SECONDARY[submission.scoreStatus as Grade], "pl-0")}>
```

Comment:
This doesn't seem to be working

Reply:
Variants got added to badge so now one can't just add a custom color, could do a custom variant on badge or just code it correctly only here since it isn't reused